.github/workflows/release.yml

```diff
@@ -50,7 +51,7 @@ jobs:
           tag=$(echo ${{ github.ref }} | cut -c11-) # get tag name without tags/refs/ prefix.
           img=$(ko build --bare --platform=all -t latest -t ${{ github.sha }} -t ${tag} ./)
           echo "built ${img}"
-          cosign sign ${img} \
+          cosign sign ${img} --yes \
```

Comment:
what do you think about using the `COSIGN_YES` variable rather than using the `--yes` flag? 🤔

Reply:
Is there a reason to prefer one over the other? I suppose if we add a second `cosign` invocation here we'd also need to remember to pass `--yes` to that as well.